js/bybit.js

```diff
@@ -48,7 +48,7 @@ module.exports = class bybit extends Exchange {
                 'fetchDepositAddressesByNetwork': true,
                 'fetchDeposits': true,
                 'fetchFundingRate': true,
-                'fetchFundingRateHistory': false,
+                'fetchFundingRateHistory': true,
                 'fetchIndexOHLCV': true,
                 'fetchLedger': true,
```

Comment:
@sc0Vu I think we're missing this ugprade: https://bybit-exchange.github.io/docs/account_asset/v3/#t-withdrawrecordquery

Reply:
Do you mean this one: https://github.com/ccxt/ccxt/blob/bd63662db44909783b7e5ee0fc9e73e3ecdbc706/js/bybit.js#L5330 ?

Reply:
@sc0Vu sorry I pasted the wrong link, I meant this one: https://bybit-exchange.github.io/docs/derivativesV3/contract/#t-dv_walletrecords